specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-09-01/examples/CosmosDBCassandraTableCreateUpdate.json

```diff
@@ -0,0 +1,79 @@
+{
+  "parameters": {
+    "accountName": "ddb1",
+    "resourceGroupName": "rg1",
+    "api-version": "2020-09-01",
+    "subscriptionId": "subid",
+    "keyspaceName": "keyspaceName",
+    "tableName": "tableName",
+    "createUpdateCassandraTableParameters": {
+      "location": "West US",
+      "tags": {},
+      "properties": {
+        "resource": {
+          "id": "tableName",
+          "defaultTtl": 100,
+          "analyticalStorageTtl": 500,
```

Comment:
please remove analyticalStorageTtl

Reply:
same for create mongo collection example

Reply:
Removed analyticalStorageTtl from all examples